mkosi

```diff
@@ -4422,7 +4424,10 @@ def build_stuff(args: CommandLineArguments) -> None:
     run_finalize_script(args, workspace.name, verb='final')
 
     # Run the image builder for the second (final) stage
-    raw, tar, root_hash = build_image(args, workspace, do_run_build_script=False, cleanup=True)
+    if args.skip_final_phase:
+        print_step('Skipping (second) final image build phase.')
+    else:
+        raw, tar, root_hash = build_image(args, workspace, do_run_build_script=False, cleanup=True)
```

Comment:
But you're not skipping the steps below... In your usecase you may also want to make sure type is set to directory.

Reply:
I'm not sure I understand your comment, but I tested the patch and it seems to work as I expected. Did I miss something?

Reply:
`raw`, `tar`, `root_squash` are used unconditionally right below. If this call to `build_image` is skipped, they might not be set at all.